jsonnet/kube-thanos/kube-thanos-compactor.libsonnet

```diff
@@ -49,7 +49,10 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           container.mixin.resources.withLimits({ cpu: '500m', memory: '2Gi' }) +
           container.withVolumeMounts([
             containerVolumeMount.new('thanos-compactor-data', '/var/thanos/compactor', false),
-          ]);
+          ])
+          +
```

Comment:
Nit: Is this `jsonnetfmt`? Maybe move the plus up a line at the end?

Reply:
Oops I've missed that one